civihr_employee_portal/features/civihr_default_permissions/civihr_default_permissions.features.user_permission.inc

```diff
@@ -1620,6 +1620,13 @@ function civihr_default_permissions_user_default_permissions() {
     'module' => 'webform',
   );
 
+  // Exported permission: 'edit system workflow message templates'.
+  $permissions['edit system workflow message templates'] = array(
+    'name' => 'edit system workflow message templates',
+    'roles' => array(),
```

Comment:
Shouldn't the administrator get this permission?

Reply:
Will be a good idea 👌